src/js/libcs/RenderBackends.js

```diff
@@ -868,3 +868,9 @@ globalInstance.exportPDF = function() {
         exportWith(PdfWriterContext, wnd);
     });
 };
+
+globalInstance.exportPNG = function() {
+    var data = csctx.canvas.toDataURL();
+    var wnd = window.open('about:blank', '_blank');
+    wnd.location.href = data;
```

Comment:
I haven't seen any problems with popup blockers if the new windows were opened in immediate response to a button event, i.e. from within the event handler. Which is the reason the other export functions create a window with `about:blank` as URL directly from within the handler, even if they only change the location of said window later in in an asynchronous way.